package.json

```diff
@@ -25,22 +25,25 @@
     "web-vitals": "^1.1.1"
   },
   "scripts": {
-    "start": "env-cmd -e development react-scripts start",
-    "build": "env-cmd -e development react-scripts build",
-    "build:stage": "env-cmd -e staging react-scripts build",
     "build:prod": "env-cmd -e production react-scripts build",
+    "build:stage": "env-cmd -e staging react-scripts build",
+    "build": "env-cmd -e development react-scripts build",
+    "cypress:open": "./node_modules/.bin/cypress open",
+    "cypress:run": "./node_modules/.bin/cypress run",
```

Comment:
**nitpick**: remove node modules path
```suggestion
    "cypress:open": "cypress open",
    "cypress:run": "cypress run",
```

**question**: since the dev server needs to be running should these also run npm start prior?

Reply:
I've gone back and forth with adding `npm start` to the cypress commands. You have to [use a separate package to run the commands concurrently](https://github.com/agrc/electrofishing/blob/f4a11f72879b325726b1580e7eb52b61e82eb7d8/package.json#L72) and it's a bit of a pain when you are developing. I decided to keep it out, in this case, to see how I liked it.